plugins/module_utils/foreman_helper.py

```diff
@@ -286,8 +303,10 @@ def __init__(self, argument_spec, **kwargs):
             state=dict(choices=['present', 'absent'], default='present'),
         )
         args.update(argument_spec)
+        name_map = kwargs.pop('name_map', {})
```

Comment:
Do we really want to touch the `nailgun` API?

Reply:
You did for the documentation. But, you are right. There is no benefit.

I thought about moving the not nailgun Classes to the cement file, because i accidentally added my changes there, and wondered, nothing happend.

Reply:
I did, because it was a no-code change ;)